examples/vite/src/index.scss

```
@@ -9,6 +9,7 @@
 @import url('./AppSettings/AppSettings.scss') layer(stream-app-overrides);
 @import url('./CustomMessageActions/CustomMessageActions.scss')
 layer(stream-app-overrides);
+@import url('./InlineEditMessage/InlineEditMessage.scss') layer(stream-app-overrides);
```

Stylelint (17.14.0)

[error] Expected "url('./InlineEditMessage/InlineEditMessage.scss')" to be "'./InlineEditMessage/InlineEditMessage.scss'" (import-notation)

(import-notation)